system/services/admin/AuditService.cfc

```diff
@@ -1,11 +1,21 @@
+/**
+ * Provides logic for interacting with form builder forms
+ * @presideservice
+ */
 component output="false" singleton=true {
 
 // CONSTRUCTOR
 	/**
 	 * @dao.inject presidecms:object:audit_log
+	 * @coldboxController.inject  coldbox
 	 */
-	public any function init( required any dao ) output=false {
+	public any function init(
+		  required any dao
+		, required any coldboxController
```

Comment:
Because this service is tagged as `@presideService`, we do not need to inject `coldboxController` here. Instead, we can just use `$getColdbox()`. See https://docs.presidecms.com/devguides/presidesuperclass.html